iam/certificate-acl/app/server/createFramework.js

```diff
@@ -0,0 +1,14 @@
+const {
+  WebUtils, Logger,
+} = require('@dojot/microservice-sdk');
+
+const logger = new Logger('certificate-acl:express');
+
+module.exports = (aclRoute) => WebUtils.framework.createExpress(
+  {
```

Comment:
Perhaps some of these middleware are interesting, with readinessInterceptor and beaconInterceptor

https://github.com/dojot/backstage/blob/ca9ffa2a205e7c09265cd28e2458bedc7f6d3580/app/express/index.js#L38

Reply:
I'm not so familiar with the webutils. Thanks for point out this!